requirements.txt

```
@@ -40,3 +40,4 @@ langchain-text-splitters
 # the below library should NOT be imported into any python files
 # it is for CLI usage ONLY
 yamllint>=1.35.1,<1.36.0
+instructlab.sdg==0.0.1rc1
```

```suggestion
instructlab-sdg==0.0.1rc1
```

Isn't the package name with a dash?

https://github.com/instructlab/sdg/blob/c46d5067fa2ee19dccb0530026b2d8109147464d/pyproject.toml#L8

---

good catch, thanks!